src/person/controllers/update_person_controller.py

```diff
@@ -46,6 +48,9 @@ def process(self):
         pending_feedback = self._data.get("pending_feedback")
         deleted = self._data.get("deleted")
         fcm_registration_token = self._data.get("fcm_registration_token")
+        is_paused = self._data.get("is_paused")
+        pause_expiration = self._data.get("pause_expiration")
```

Comment:
this allows the client to set `pause_expiration`, which we don't want to happen. change this to 
```python
pause_expiration = None
```